test/new-e2e/system-probe/connector/main.go

```diff
@@ -44,6 +44,7 @@ type args struct {
 	serverKeepAliveMaxCount int
 	sshFilePath             string
 	vmCommand               string
+	allowAPIKey             bool
```

Comment:
Can we make this a list of environment variables we would like to forward? This way we will not fall into this trap with something else.

Reply:
Changed!